optimum/onnxruntime/modeling_ort.py

```diff
@@ -833,3 +781,39 @@ def forward(
         return ImageClassifierOutput(
             logits=torch.from_numpy(outputs[self.model_outputs["logits"]]),
         )
+
+
+class ORTModelForCustomTasks(ORTModel):
```

Comment:
i like it!

Reply:
Cool!! I will make it neater, it's a draft idea right now.

Reply:
Awesome! would you mind adding similar documentation to what we have for the others? Maybe you can be creative with a non-supported task/model, but which would work.

Reply:
Example added!